src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -107,10 +106,8 @@ def validate_import_key(key):
 
 def validate_import_feature(feature):
     if feature:
-        invalid_pattern = re.compile(r'[^a-zA-Z0-9._-]')
-        invalid = re.search(invalid_pattern, feature)
-        if invalid:
-            logger.warning("Ignoring invalid feature '%s'. Only alphanumeric characters, '.', '-' and '_' are allowed in feature name.", feature)
+        if '%' in feature:
+            logger.warning("Ignoring invalid feature '%s'. Feature name cannot contain the '%%' character.", feature)
```

Comment:
>'%%' [](http://example.com/codeflow?start=91&length=4)

one %?

Reply:
We have to escape the % with another % here. The warning message will contain only one %.

Reply:
Got it. It is used to escape the % place holder.